### ISSUE TITLE
Doesn't show translator comments

### ISSUE THREAD

**Issue by murraycu** [delight-im/Localize#14]
I noticed that my XML comments for translators are not visible in the list of phrases.

**Comment by boris-dev** [comment]
I agree, that's the best way to manage comments for translations. Does It work now?

**Comment by mwaclawek** [comment]
No, not yet, unfortunately. But we should add it together with https://github.com/delight-im/Localize/issues/20
